test/faker/internet_test.exs

```diff
@@ -37,4 +41,44 @@ defmodule Faker.InternetTest do
 
     assert Regex.match?(~r/\A((hello|world)-)(hello|world)\z/, slug)
   end
+
+  test "user_name/0" do
+    Stream.repeatedly(&user_name/0)
+    |> Enum.take(@iterations)
+    |> Enum.each(fn generated_value ->
+      assert Regex.match?(~r/^((?!").)|((?!').)*$/, generated_value)
```

Comment:
how about `refute`? So we are actually checking that we don't have quotes? like:

```
refute String.contains?(generated_value, ~s('"))
```